xls/examples/klayout.lyt

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think the actual upstream is https://github.com/efabless/sky130_klayout_pdk/blob/main/sky130_tech/tech/sky130/sky130.lyt

Reply:
This is not a part of this PR anymore. It was added here only for testing purpose on the early stage of development.
It could be removed because KLayout tech files can now be supplied in `bazel_rules_hdl` with https://github.com/hdl/bazel_rules_hdl/commit/5e2ec27916ed3a5b956f76d63e632c24b8bd1f81